CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
-## 0.10.0
+## 0.11.0
+
+* Added `span` option where the tcp events will be logged. The interface matches the opentracing span interface.
+* Improved retries performance and memory usage.
```

Comment:
Do we actually have some numbers for the retries claim?

Reply:
No, but I am copying the claim of the circuit breaker changes which are similar. :smile: